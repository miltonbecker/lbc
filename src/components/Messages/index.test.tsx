import React from 'react';
import renderer from 'react-test-renderer';
import { Messages } from './';

describe('Messages', () => {
  it('Renders correctly', () => {
    const tree = renderer
      .create(
        <Messages />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
