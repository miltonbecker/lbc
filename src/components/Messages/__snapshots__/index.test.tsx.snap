// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Messages Renders correctly 1`] = `
<div
  className="container"
>
  <section
    className="header"
  >
    <h1>
      Messages
    </h1>
    <button
      disabled={false}
      onClick={[Function]}
    >
      Log in
    </button>
    <button
      disabled={true}
      onClick={[Function]}
    >
      Log out
    </button>
  </section>
  <div
    className="post"
  >
    <h2>
      Add message
    </h2>
    <form
      className="form"
      onSubmit={[Function]}
    >
      <textarea
        cols="50"
        rows="6"
      />
      <button>
        Post
      </button>
    </form>
  </div>
  <p>
    Fetching messages...
  </p>
</div>
`;
